form_to_clean_csv.py

```diff
@@ -0,0 +1,23 @@
+import pandas as pd
+import sys
```

Comment:
This should be

```python
import sys

import pandas as pd
```

to comply with PEP8

Reply:
We should also add pandas to `requirements.txt`

Reply:
i added pandas to the requirements, can you check if the way i added is accurate please